tpr/tpr.go

```diff
@@ -263,13 +263,36 @@ func (t *TPR) create() error {
 		Description: t.description,
 	}
 
-	_, err := t.k8sClient.ExtensionsV1beta1().ThirdPartyResources().Create(tpr)
+	createTpr := func() error {
+		_, err := t.k8sClient.ExtensionsV1beta1().ThirdPartyResources().Create(tpr)
+		if err != nil && apierrors.IsAlreadyExists(err) {
+			return microerror.Mask(alreadyExistsError)
+		}
+		if err != nil {
+			return microerror.Maskf(err, "creating TPR %s", t.name)
+		}
+
+		return nil
+	}
+
+	// Try creating the TPR once without a retry, since I
+	// do not want to retry on the 'alreadyExistsError'.
```

Comment:
You have PermanentError for that https://github.com/cenkalti/backoff/blob/309aa717adbf351e92864cbedf9cca0b769a4b5a/retry.go#L74

Reply:
I think you'll have to switch backoff version to master in glide.yaml. Which should be done anyway.